src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -437,7 +437,7 @@ JNI_METHOD(void, updateDevice)(JNIEnv * env, jobject self, jlong handle, jlong f
     StackLockGuard lock(JniReferences::GetInstance().GetStackLock());
 
     AndroidDeviceControllerWrapper * wrapper = AndroidDeviceControllerWrapper::FromJNIHandle(handle);
-    CHIP_ERROR err = wrapper->Controller()->UpdateDevice(static_cast<chip::NodeId>(deviceId), static_cast<uint64_t>(fabricId));
+    CHIP_ERROR err                           = wrapper->Controller()->UpdateDevice(static_cast<chip::NodeId>(deviceId));
```

Comment:
These aligned equals are starting to really make me consider commenting on our style rules. Extending a line width that much doesn't help :( Not an issue for this PR, but we should consider not doing "="-aligment.

Reply:
For this PR, I can add a blank line, and that should remove this alignment change.